rosdep/base.yaml

```
@@ -4418,6 +4418,7 @@ libopenni2-dev:
   fedora: [openni-devel]
   gentoo: [dev-libs/OpenNI2]
   nixos: [openni2]
+  rhel: [libfreenect-openni]
```

So it looks like this is only available in EPEL 8.  Therefore:

```suggestion
  rhel:
    '*': [libfreenect-openni]
    '7': null
```